src/components/BottomBar/Origin/NodeNavigationButton.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {
+  MdCenterFocusStrong,
+  MdFlashOn,
+  MdFlight
+} from 'react-icons/md';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import Focus from 'svg-react-loader?name=Focus!../../../icons/focus.svg';
+
+import { NavigationAimKey, NavigationAnchorKey, RetargetAnchorKey } from '../../../api/keys';
+import InfoBox from '../../common/InfoBox/InfoBox';
+import Button from '../../common/Input/Button/Button';
+import Row from '../../common/Row/Row';
+import SvgIcon from '../../common/SvgIcon/SvgIcon';
+
+import styles from './NodeNavigationButton.scss';
+
+const NavigationTypes = {
+  jump: 'jump',
+  fly: 'fly',
+  focus: 'focus',
+  frame: 'frame'
+};
+
+function NavigationButton({
+  identifier, onFinish, showLabel, type
+}) {
+  const luaApi = useSelector((state) => state.luaApi);
+
+  function focus(event) {
+    if (!event.shiftKey) {
+      luaApi.setPropertyValueSingle(RetargetAnchorKey, null);
```

Comment:
I don't know which is best but just noted the change to the lua api from the `propertyDispatcher(dispatch, NavigationAnchorKey).set(identifier);`. Is there a reason for changing the call?

Reply:
I changed it for consistency  Since a script is being called for the other buttons, it seemed more consistent that it would be so for the anchor change / refocusing as well. That way, the behavior is the same when the user presses any of the buttons (i.e. the corresponding script for that change shows up in the script log)